trpc/queries/goals.ts

```diff
@@ -341,6 +341,9 @@ export const goalDeepQuery = {
                 },
             },
         },
+        where: {
+            archived: { not: true },
```

Comment:
you can use `const notArchived = { archived: { not: true } }`

and then use query like `where: notArchived` which is both DRY and looks clean